dolphinscheduler-worker/src/main/java/org/apache/dolphinscheduler/server/worker/runner/TaskExecuteThread.java

```diff
@@ -278,8 +280,14 @@ public void downloadResource(String execLocalPath, Logger logger, List<Pair<Stri
                 String tenantCode = fileDownload.getRight();
                 String resHdfsPath = storageOperate.getResourceFileName(tenantCode, fullName);
                 logger.info("get resource file from hdfs :{}", resHdfsPath);
+                long resourceDownloadStartTime = System.currentTimeMillis();
                 storageOperate.download(tenantCode, resHdfsPath, execLocalPath + File.separator + fullName, false, true);
+                WorkerServerMetrics.recordWorkerResourceDownloadTime(System.currentTimeMillis() - resourceDownloadStartTime);
+                WorkerServerMetrics.recordWorkerResourceDownloadSize(
+                        FileUtils.getFileSizeInByte(execLocalPath + File.separator + fullName));
```

Comment:
```suggestion
                WorkerServerMetrics.recordWorkerResourceDownloadSize(
                        Files.size(Paths.get(execLocalPath, fullName)));
```

Reply:
Pretty and neat! Have fixed it in the latest commit. Thx